Transparency in PlotPosteriorComparison

wanjam opened the pull request:
- FaceAlpha is managed in a different way starting Matlab R2014b
- This checks the version and uses the appropriate way
- Also: PlotPosteriorComparison can now take an optional input argument to plot condition-labels instead of numbers.

Commit:
Transparency in PlotPosteriorComparison

* FaceAlpha is managed in a different way starting Matlab R2014b
* This checks the version and uses the appropriate way
* PlotPosteriorComparison can now take an optional input argument to plot condition-labels instead of numbers.'

suchow commented:
Hi, thanks for the contribution! I have one question for you: 

On lines 15 to 20, you renamed `PlotPosteriorComparison` to `PlotPosteriorComparisonWanja`, which I think will break existing code that uses the original name. Any reason for the renaming?

Commit:
Corrected typo in PlotPosteriorComparison and adjusted ColorWorkingMemoryExperiment for newer Matlab versions

* Corrected the name to PlotPosteriorComparison
* ColorWorkingMemoryExperiment was still using RandSample, which is called PsychRandSample in newer versions of Psychtoolbox
* ColorWorkingMemoryExperiment was throwing an error because 'buttons' was not defined
* ColorWorkingMemoryExperiment still used RandStream.setDefaultStream which was renamed to RandStream.setGlobalStream
	* see https://de.mathworks.com/help/matlab/release-notes.html?rntext=RandStream&startrelease=R2010b&endrelease=R2011a&groupby=release&sortby=descending&searchHighlight=RandStream

wanjam commented:
Hi,

No! That was just a leftover from testing. Sorry for the mess, I pushed a new version. 
I also adjusted ColorWorkingMemoryExperiment.m, which should fix #29 .
